
rec_imu : True
rec_rgb : False
rec_depth : True
rec_tf : True
rec_confidence: False